### PR TITLE
Dpl-998-2 UI Cleanup of aliquot work

### LINKED ISSUE #1605: DPL-998-2 - As developer we need to tidy up after the Revio multiplexing work to remove any code, data and tables that are no longer needed to reduce technical debt and improve maintainability
**Describe the Housekeeping**
As developer we need to tidy up after the Revio multiplexing work to remove any code, feature flags, components and tests that are no longer needed to reduce technical debt and improve maintainability

**Blocking issues**
Describe any other issues or tickets that may be blocking this change.

Things to do:

- [ ]  Remove components/pacbio/V1 folder
- [ ]  Remove tests/unit/components/pacbio/V1 folder
- [ ]  Remove all files suffixed with V1
    - src/views
    - tests/unit /views
    - tests/data
    - tests/e2e/specs/pacbio
    - tests/fixtures
- [ ]  Remove code that handles the unintended data returned from service in pacbioPoolCreate
- [ ]  Remove following feature flags
    - dpl_989_ui
    - multiplexing_phase_2_pool_with_aliquot
    - dpl_1113_run_well_aliquots
    - dpl_1113_run_edit_well_aliquots

- [ ] Review the way used_aliquot data is pulled back in run create. Now we have the well relationships libraries and pools through used_aliquots in the service we may no longer need to format the returned wells. We can also pull back tubes with the main query instead of 2 separate queries.

### PR DESCRIPTION
**Pull request by seenanair**
Closes #1605 

#### Changes proposed in this pull request
- Removes all source files with V1 suffix which uses stores with non-aliquot data  
   -  src/compoments/pacbio/V1 folder
   -  src/views
   -  src/components/labware
- Removes feature flags
 dpl_1112,multiplexing_phase_2_aliquot,dpl_989_ui,multiplexing_phase_2_pool_with_aliquots,dpl_1113_run_well_aliquots ,dpl_1113_run_edit_well_aliquots 

- Removes all stores that support non-aliquot data  
   src/store/pacbio/pacbioPoolCreate
  src/stores/pacbioRunCreateV1

- Remove all unit and e2e test cases for component, views ans store with non-aliquot informatiom
- Refactored the way used_aliquots is pulled back in run create store to fetch wells, requests, pools, libraries
    - Refactored setRun so that all data including tubes and request are pulled back using one query( fetchRun )
    - Removed code (formatWells) that manually sets pool and library relations in well
    
- Refactored the way used_aliquots is pulled back in pool create to fetch plates, libraries and tubes
   - Removed unnecessary code that handles redundant data returned by service

### CODE REVIEW

File: src/stores/pacbioRunCreate.js [hunk 1]
```diff
@@ -359,9 +313,8 @@ export const usePacbioRunCreateStore = defineStore('pacbioRunCreate', {
       const request = rootStore.api.traction.pacbio.runs
       const promise = request.find({
         id,
-        // The used aliquots are needed to get the libraries, pools and tubes
         include:
-          'plates.wells.used_aliquots.library.tube,plates.wells.used_aliquots.pool.tube,smrt_link_version',
+          'plates.wells.used_aliquots,plates.wells.libraries.tube,plates.wells.pools.tube,plates.wells.libraries.request,plates.wells.pools.requests,plates.wells.pools.used_aliquots,plates.wells.libraries.used_aliquots,smrt_link_version',
```
Comment (stevieing) [inline src/stores/pacbioRunCreate.js:317]: Should the tags be in here?

**Review by stevieing — CHANGES_REQUESTED** [review]

File: src/stores/pacbioPoolCreate.js
Comment (BenTopping) [inline src/stores/pacbioPoolCreate.js:1]: Lines 22-28 can be removed I think (commented out old function)

File: src/stores/pacbioPoolCreate.js [hunk 2]
```diff
@@ -601,7 +601,7 @@ export const usePacbioPoolCreateStore = defineStore('pacbioPoolCreate', {
         // after the request for all plates, as otherwise the partial record will over-write
```
Comment (BenTopping) [inline src/stores/pacbioPoolCreate.js:601]: Can we remove the comment block above this now that the correct records are loaded?

**Review by BenTopping — COMMENTED** [review]
Really good. 25k lines removed 👏

File: src/stores/pacbioRunCreate.js (hunk 1 above)
Comment (seenanair) [reply src/stores/pacbioRunCreate.js]: Very good finding 👍 Updated with tags

File: src/stores/pacbioPoolCreate.js (hunk 2 above)
Comment (seenanair) [reply src/stores/pacbioPoolCreate.js]: Done 👍

**Review by stevieing — APPROVED** [review]

**Review by BenTopping — APPROVED** [review]

**Pull request merged**